### ISSUE TITLE
No `::-webkit-input-placeholder` in output

### ISSUE THREAD

**Issue by ArmorDarks** [giakki/uncss#199]
For some reason `::-webkit-input-placeholder` doesn't end up in output:

``` css
::-webkit-input-placeholder { font-size: 18px; }
:-moz-placeholder { font-size: 18px; }
::-moz-placeholder { font-size: 18px; }
:-ms-input-placeholder { font-size: 18px; }
```

results in

``` css
:-moz-placeholder { font-size: 18px; }
::-moz-placeholder { font-size: 18px; }
:-ms-input-placeholder { font-size: 18px; }
```

Maybe somehow related to https://github.com/giakki/uncss/issues/79?

**Comment by XhmikosR** [comment]
Should be fixed by #211.

**Comment by ArmorDarks** [comment]
Nice, thanks!
